core/tests/protractor.conf.js

```diff
@@ -44,7 +44,7 @@ var suites = {
     explorationFeedbackTab: [
       'protractor_desktop/explorationFeedbackTab.js'
     ],
-
+    
```

Comment:
This is a stray change. Please remove it

Reply:
Removed.